certified-connectors/YakChat/apiDefinition.swagger.json

```diff
@@ -0,0 +1,382 @@
+        ]
+      }
+    },
+    "/Automation/DynamicInboxes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "response",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "$values": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "DisplayName": {
+                        "type": "string",
+                        "description": "The display name of the option",
+                        "title": "Display Name"
+                      },
+                      "Value": {
+                        "type": "string",
+                        "description": "The value of the option",
+                        "title": "Value"
+                      }
+                    }
+                  },
+                  "description": "$values"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Dynamic Inboxes",
+        "description": "(Private) Gets the YakChat Inboxes for a user to populate dynamic dropdowns",
+        "operationId": "GetDynamicInboxes",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/Automation/InboundMessageNotification": {
+      "x-ms-notification-content": {
+        "description": "Incoming message event",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "Direction": {
+              "type": "string",
+              "description": "Whether the message is inbound or outbound",
+              "title": "Direction"
+            },
+            "ToNumber": {
+              "type": "string",
+              "description": "The number the message was sent to",
+              "title": "To"
+            },
+            "FromNumber": {
+              "type": "string",
+              "description": "The number from which the message was sent",
+              "title": "From"
+            },
+            "Text": {
+              "type": "string",
+              "description": "The text of the message",
+              "title": "Text"
+            },
+            "MediaUrl": {
+              "type": "string",
+              "description": "The URL for any attached media, if the message was an MMS message",
+              "title": "Message Media URL"
+            },
+            "TimeStamp": {
+              "type": "string",
+              "description": "Timestamp of when the message was receieved",
+              "title": "Time Stamp"
+            },
+            "InboxEmail": {
+              "type": "string",
+              "description": "The YakChat inbox that received the message",
+              "title": "Inbox"
+            }
+          }
+        }
+      },
+      "post": {
+        "responses": {
+          "200": {
+            "description": "response",
+            "schema": {}

```

Comment:
@aMatthewsYakChat Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Applies the comment for all the similar empty schema definitions.

Reply:
This has been updated